src/WebJobs.Script/Binding/Http/HttpTriggerAttribute.cs

```diff
@@ -10,8 +12,36 @@ public sealed class HttpTriggerAttribute : Attribute
     {
         public HttpTriggerAttribute()
         {
+            AuthLevel = AuthorizationLevel.Function;
         }
 
-        public string RouteTemplate { get; set; }
+        public HttpTriggerAttribute(AuthorizationLevel authLevel, params string[] methods)
+        {
+            AuthLevel = authLevel;
+            Methods = methods;
+        }
+
+        /// <summary>
+        /// Gets or sets the route template for the function. Can include
+        /// route parameters using WebApi supported syntax. If not specified,
+        /// will default to the function name.
+        /// See: https://www.asp.net/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2#constraints
+        /// </summary>
+        public string Route { get; set; }
```

Comment:
As part of this, I also renamed this to match the existing function.json property